server/src/com/cloud/storage/VolumeApiServiceImpl.java

```diff
@@ -1924,6 +1935,23 @@ public Volume migrateVolume(MigrateVolumeCmd cmd) {
                                 throw new InvalidParameterValueException("Cannot migrate ROOT volume of a stopped VM to a storage pool in a different VMware datacenter");
                             }
                         }
+
```

Comment:
@bhaisaab I have to agree with @rafaelweingartner. Create methods is also a great way to improve the code (no just a way to get a reusable code). It helps on keep it readable, ensure its correct implementation by implementing tests, and allows to create Javadoc. I think that it could improve this code too.
Thanks.